src/controllers/tripsController.js

```diff
@@ -308,4 +310,37 @@ export default class requestController {
       return Response.errorResponse(res, 500, err.message);
     }
   }
+
+  /**
+     * @description request feature
+     * @static
+     * @param {Object} req
+     * @param {Object} res
+     * @returns {Object} array of request
+     * @memberof requestController
+     */
+  static async requestSearch(req, res) {
+    try {
+      const { user } = req;
+      const {
+        id, location, destination, status, reason, departureDate, returnDate
+      } = req.query;
+      const field = {
+        id,
+        location,
+        destination,
+        status,
+        reason,
+        departureDate,
+        returnDate
+      };
+      const requests = await TripsService.searchRequest(field, user);
+      if (requests === 'request not found') {
```

Comment:
if `request not found` is a constant message, it should be added to string helpers (like `Helper.RequestNotFound`) to avoid cases where you could forget a space or type `Request` instead of `request` and the `if` statement fails.